samples/wifi/provisioning/softap/prj.conf

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Optimize TF-M
+CONFIG_TFM_PROFILE_TYPE_SMALL=y
+CONFIG_PM_PARTITION_SIZE_TFM_SRAM=0xa000
+CONFIG_PM_PARTITION_SIZE_TFM=0x1c000
+
+# Optimize Wi-Fi stack to save some memory.
+CONFIG_HEAP_MEM_POOL_SIZE=81920
+CONFIG_NRF700X_RX_NUM_BUFS=16
+CONFIG_NRF700X_MAX_TX_AGGREGATION=4
+
+# Wi-Fi
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+CONFIG_NRF700X_AP_MODE=y
+CONFIG_WPA_SUPP=y
+CONFIG_WPA_SUPP_AP=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_WIFI_CREDENTIALS=y
+
+# General
+CONFIG_LOG=y
+CONFIG_LOG_MODE_IMMEDIATE=y
+CONFIG_ASSERT=y
+CONFIG_ASSERT_VERBOSE=n
+
+# Link-time optimizations
+CONFIG_LTO=y
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+
+# Temporarily enable FS support so that linking succeeds.
+CONFIG_FILE_SYSTEM=y
+CONFIG_MAIN_STACK_SIZE=6144
+CONFIG_NANOPB=y
+CONFIG_SMF=y
+
+# HTTP parser
+CONFIG_HTTP_PARSER=y
+CONFIG_HTTP_PARSER_URL=y
+
+# DK Butons and LEDs
+CONFIG_DK_LIBRARY=y
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_CONNECTION_MANAGER=y
+CONFIG_NET_IPV4=y
+# Disable IPv6 to save memory
+CONFIG_NET_IPV6=n
+CONFIG_NET_TCP=y
+CONFIG_NET_UDP=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_MAX_CONN=6
+CONFIG_POSIX_MAX_FDS=20
+CONFIG_NET_SOCKETS_TLS_MAX_CONTEXTS=2
+CONFIG_NET_SOCKETS_SOCKOPT_TLS=y
+
+# Zephyr NET Connection Manager connectivity layer
+CONFIG_L2_WIFI_CONNECTIVITY=y
+CONFIG_L2_WIFI_CONNECTIVITY_AUTO_CONNECT=n
+CONFIG_L2_WIFI_CONNECTIVITY_AUTO_DOWN=n
+
+# SoftAP Wi-Fi Provision library
+CONFIG_SOFTAP_WIFI_PROVISION=y
+# Disable socket close on completed provisioning to keep mDNS SD running.
+# SD depends on the socket being open for a specified port.
+CONFIG_SOFTAP_WIFI_PROVISION_SOCKET_CLOSE_ON_COMPLETION=n
+CONFIG_SOFTAP_WIFI_PROVISION_LOG_LEVEL_DBG=y
+
+# (M)DNS
+CONFIG_MDNS_RESPONDER=y
+CONFIG_DNS_SD=y
+CONFIG_MDNS_RESPONDER_DNS_SD=y
+CONFIG_NET_HOSTNAME_ENABLE=y
+CONFIG_NET_HOSTNAME="wifiprov"
+# Disable POSIX uname support to suppress build warning that hostname is too long.
+CONFIG_POSIX_UNAME=n
+
+# DHCPv4 client and server
+CONFIG_NET_DHCPV4=y
+CONFIG_NET_DHCPV4_SERVER=y
+
+# MBed TLS heap size, required by WPA supplicant and MBed TLS.
+CONFIG_MBEDTLS_HEAP_SIZE=81920
```

Comment:
Do we really need that much? Are we using TLS connections in the sample? No objection, of course, if we do have the memory. Just wondering.

Reply:
Last time I checked, yes. But Ill have another go just to be sure. I Don't remember if I tested again after starting to use ECDSA server cert.